server/chat-plugins/chatlog.ts

```diff
@@ -660,6 +665,8 @@ export const commands: ChatCommands = {
 		"/searchlogs [search], [room], [cap], [date] - searches logs in the current room for [search].",
 		"A comma can be used to search for multiple words in a single line - in the format arg1, arg2, etc.",
 		"If a [cap] is given, limits it to only that many lines. Defaults to 500.",
+		"The delimiter | can be used to space searching for multiple terms.",
+		"Date formatting is ISO formatting (YYYY-MMMM-DDDD.) E.g 2020-05, 2020, or `all`.",
```

Comment:
```suggestion
		"Date formatting is ISO formatting (YYYY-MM-DD.) E.g 2020-05, 2020, or `all`.",
```

Months and days only have 2 digits :P